### PR TITLE
add prod rosa jobs and suspend 4.16 jobs

### PR DESCRIPTION
**Pull request by yasun1**
[OCM-5382](https://issues.redhat.com/browse/OCM-5382)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-advanced-prod-critical-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hcp-private-link-prod-critical-e2e-f7 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-advanced-prod-critical-e2e-f3 periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-private-link-prod-critical-e2e-f7

**Comment by yasun1** [comment]
@liangxia Please help review. Thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -911,7 +911,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-non-sts
-- as: aws-rosa-non-sts-advanced-f7
+- as: aws-rosa-non-sts-advanced-f360
   cron: 20 15 3,10,17,26 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:915]: Missing update for the cron ?

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:915]: yes, missed. Let me update.

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Updated. Please review again.

**Pull request merged**